### PR TITLE
Knapsack Problem using Greedy Method (WoC)

### PR DESCRIPTION
**Pull request by k-sashank**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)
Yes

### Description

(Write your answer here.)
The Knapsack Problem is a pretty prominent problem in DSA and Competitive Programming. In this problem, there a bunch of objects with different weights and different profits. There is a limit on the weight which can be included in a Knapsack. The answer must be the highest profit that can be made by including a combination of objects from the ones mentioned in the problem while following the weight constraint.

### Checklist

- [Y] I've read the contribution guidelines.
- [Y] I've checked the issue list before deciding what to submit.
- [Y] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)
Knapsack Problem - Greedy Method (WoC) #1469

### CODE REVIEW

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@k-sashank Add sample input-output. Also, Format your code according to PEP 8 rules. You can refer to this http://pep8online.com/.

**Comment by k-sashank** [comment]
@iamrajiv I have made the required changes!
Please review once again!

**Review by iamrajiv — APPROVED** [review]

**Comment by k-sashank** [comment]
@atarax665 Please review the PR!

File: Python/cp/Knapsack_Problem-Greedy_Method.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
+def Knapsack(p, w, TW):
```
Comment (HarshCasper) [inline Python/cp/Knapsack_Problem-Greedy_Method.py]: Add an Introduction here about the Algorithm

File: Python/cp/Knapsack_Problem-Greedy_Method.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
+def Knapsack(p, w, TW):
+    l = []
+    pr = []
```
Comment (HarshCasper) [inline Python/cp/Knapsack_Problem-Greedy_Method.py]: Use meaningful Variable Names. Not something ambiguous

File: Python/cp/Knapsack_Problem-Greedy_Method.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+            pr.append(l[i])
+        l[i] = -1
+    total_prof = 0
+    # Calculating total profit
+    for i in range(len(pr)):
+        total_prof += pr[i] * wt[i]
+    return total_prof
+
+# Profits
+p = [10, 5, 15, 7, 6, 18, 3]
```
Comment (HarshCasper) [inline Python/cp/Knapsack_Problem-Greedy_Method.py]: Make the input dynamic. Not a Driver Code

File: Python/cp/Knapsack_Problem-Greedy_Method.py [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+# Weights
+w = [2, 3, 5, 7, 1, 4, 1]
+# Total Weight the Knapsack can handle
+TW = 15
+x = Knapsack(p, w, TW)
+print(x)
+
+# Sample Input 1:
```
Comment (HarshCasper) [inline Python/cp/Knapsack_Problem-Greedy_Method.py]: Give sample I/O in a Multi-Line Comment and not Single-Line

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Comment by k-sashank** [comment]
@HarshCasper I made the required changes!
Please review again!

**Comment by k-sashank** [comment]
@HarshCasper I made the required changes!
Please review again!

File: Python/cp/Knapsack_Problem-Greedy_Method.py [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+for i in range(n):
+    weights.append(int(input()))
+
+#Total Weight Limit (Constraint)
+print("Enter the total weight limit: ")
+Total_Weight_Limit = int(input())
+
+#Output
+total_profit = Knapsack(profits, weights, Total_Weight_Limit)
+print(total_profit)
```
Comment (HarshCasper) [inline Python/cp/Knapsack_Problem-Greedy_Method.py:60]: Enclose this in a `if __name__ == '__main__'` Block.

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Resolve the Merge Conflicts as well. Rest looks good 👍

**Comment by HarshCasper** [comment]
Are you still working on it @k-sashank ?

**Pull request closed without merging**